decktape.js

```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headed : {
```

Comment:
I'd suggest `headless`, WDYT?

Reply:
I agree, that seems to better align with the puppeteer API